semgrep-core/old/lang_php/spatch_php.ml

```diff
@@ -143,7 +143,9 @@ let parse file =
   toks
   |> List.iter (fun tok ->
          let line = Parse_info.line_of_info tok in
-         let annot = Hashtbl.find hline_env line in
+         let annot =
+           try Hashtbl.find hline_env line with Not_found -> raise Not_found
```

Comment:
This line looks strange. I'm guessing it's a left-over from debugging?

Reply:
no it's to remove the semgrep warning about the use of Hashtbl.find :)

Reply:
Aha! Assuming that there is no `try-with` that catches it, we could use a `failwith` (if we expect it to happen) or log an error (if we don't).